docs/pages/enroll-resources/machine-id/workload-identity/getting-started.mdx

```diff
@@ -123,6 +123,22 @@ services:
       - 10.0.0.1
 ```
 
+You will also need to modify the Role to explicitly grant the Bot permission to
+request an SVID including a DNS or IP SAN. For example:
+
+```yaml
+kind: role
+version: v6
+metadata:
+  name: spiffe-issuer
+spec:
+  allow:
+    spiffe:
+    - path: "/svc/foo"
+      dns_sans: ["example.com"]
+      ip_sans: ["10.0.0.1/32"]
```

Comment:
I would make it more explicit which fields a user needs to configure. The sentence before the snippet says that permissions should include a DNS or IP SAN, but the configuration example includes both `dns_sans` and `ip_sans`.

Reply:
[adfb315](https://github.com/gravitational/teleport/pull/45432/commits/adfb315ab18a796a23b8daa56eb59d2c12e707fc)